client/extensions/woocommerce/app/dashboard/setup.js

```diff
@@ -0,0 +1,196 @@
+					{
+						label: translate( 'Import' ),
+						path: this.itemLink( '/store/products/:site/import' ),
+						slug: 'add-products-import',
+					},
+					{
+						label: 'Add a product',
+						path: this.itemLink( '/store/products/:site/add' ),
+					}
+				]
+			},
+			{
+				checked: storeShippingIsSetUp,
+				docURL: 'https://support.wordpress.com/',
+				explanation: translate( 'Configure the locations to which you ship your products.' ),
+				label: translate( 'Set up shipping' ),
+				show: this.state.showShippingTask,
+				actions: [
+					{
+						label: translate( 'Set up shipping' ),
+						path: this.itemLink( '/store/settings/:site/shipping' ),
+					},
+					{
+						label: translate( 'I won\'t be shipping' ),
+						isSecondary: true,
+						onClick: this.onClickNoShip
+					}
+				]
+			},
+			{
+				checked: storePaymentsAreSetUp,
+				docURL: 'https://support.wordpress.com/',
+				explanation: translate( 'Choose which payment methods to offer your customers.' ),
+				label: translate( 'Set up payments' ),
+				show: true,
+				actions: [
+					{
+						label: translate( 'Set up payments' ),
+						path: this.itemLink( '/store/settings/:site/payments' ),
+					}
+				]
+			},
+			{
+				checked: storeTaxesAreSetUp,
+				docURL: 'https://support.wordpress.com/',
+				explanation: translate( 'Configure how tax rates are calculated at your store.' ),
+				label: translate( 'Set up taxes' ),
+				show: this.state.showTaxesTask,
+				actions: [
+					{
+						label: translate( 'Set up taxes' ),
+						path: this.itemLink( '/store/settings/:site/tax' ),
+					},
+					{
+						label: translate( 'I\'m not charging sales tax' ),
+						isSecondary: true,
+						onClick: this.onClickNoTaxes
+					}
+				]
+			},
+			{
+				checked: storeHasBeenCustomized,
+				docURL: 'https://support.wordpress.com/',
+				explanation: translate( 'View your store, test your settings and customize the design.' ),
+				label: translate( 'View and customize' ),
+				show: true,
+				actions: [
+					{
+						label: translate( 'Customize' ),
+						path: this.itemLink( 'https://:site/wp-admin/customize.php?return=%2Fwp-admin%2F' ),
+					}
+				]
+			}
+		];
+	}
+
+	renderSetupTask = ( setupTask, index ) => {
+		if ( ! setupTask.show ) {
+			return null;
+		}
+
+		return (
+			<SetupTask
+				actions={ setupTask.actions }
+				checked={ setupTask.checked }
+				docURL= { setupTask.docURL }
+				explanation={ setupTask.explanation }
+				key={ index }
+				label={ setupTask.label }
+			/>
+		);
+	}
+
+	render = () => {
+		const { onFinished, translate } = this.props;
+
+		return (
+			<div>
+				<SetupHeader
+					imageSource={ '/calypso/images/extensions/woocommerce/woocommerce-setup.svg' }
+					title={ translate( 'Howdy! Let\'s set up your store & start selling' ) }
+					subtitle={ translate( 'Below you will find the essential tasks to complete before making your store live.' ) }
+				/>
+				{
+					this.getSetupTasks().map( this.renderSetupTask )
```

Comment:
Since this is a one liner, I think we should have it on the same line as the brackets.